File: Modul_2/Soal_5/PRAK205-2310817210015-MuhammadBukhariFitri.py
import math as m

userInput = input().split()

# Test Case 1
if len(userInput) == 2:
    a, b = map(int, userInput)

# Test Case 2
elif len(userInput) == 1:
    a = int(userInput[0])
    b = int(input())

tinggi = a
sisiMiring = b
alas = m.sqrt((sisiMiring * sisiMiring) - (tinggi * tinggi))
keliling = alas + tinggi + sisiMiring
luas = (alas * tinggi) / 2

print(f"Alas = {alas:.0f} cm")
print(f"Tinggi = {tinggi} cm")
print(f"Keliling = {keliling:.0f} cm")
print(f"Luas = {luas:.0f} cm^2")

File: Modul_2/Soal_5/PRAK205-2310817210015-MuhammadBukhariFitri.c
#include <stdio.h>
#include <math.h>

int main() {
    int a;
    int b;
    scanf("%d", &a);
    scanf("%d", &b);

    int tinggi = a;
    int sisiMiring = b;
    int alas = sqrt((sisiMiring * sisiMiring) - (tinggi * tinggi));
    int keliling = alas + tinggi + sisiMiring;
    int luas = (alas * tinggi) / 2;

    printf("Alas = %d cm\n", alas);
    printf("Tinggi = %d cm\n", tinggi);
    printf("Keliling = %d cm\n", keliling);
    printf("Luas = %d cm^2", luas);
}